js/interface.js

```diff
@@ -69,6 +73,10 @@ var definitionEditor = CodeMirror.fromTextArea($('#definition')[0], {
 
 var emptyColumnNameRegex = /^Column\s\([0-9]+\)$/;
 
+Fliplet.App.Tokens.get({ appId: widgetData.appId, query: { type: 'integrationToken', order: 'createdAt', direction: 'DESC' } }).then(function(tokens) {
+  integrationTokenList  = tokens;
```

Comment:
Please explain why ESLint did not catch this.

```suggestion
  integrationTokenList = tokens;
```

Reply:
It's my bad. while checking the eslint issue with version 1.0.0 and 0.8.0 by mistake added this. I will take care this onwards.